src/index.js

```diff
@@ -141,15 +158,19 @@ async function getRDSValue(key, awsParameters) {
 }
 
 const AWS_HANDLERS = {
+  ecs: getECSValue,
   ess: getESSValue,
   kinesis: getKinesisValue,
   dynamodb: getDynamoDbValue,
   rds: getRDSValue,
   ec2: getEC2Value
 }
 
-const DEFAULT_AWS_PATTERN = /^aws:\w+:[\w-.]+:[\w.]+$/
-const SUB_SERVICE_AWS_PATTERN = /^aws:\w+:\w+:[\w-.]+:[\w.]+$/
+/* eslint-disable no-useless-escape */
```

Comment:
What is the useless escape for?

Reply:
eslint thinks that the escaped [] is useless. Without the escape the regex does not compile. Seems like a false positive to me.